Rust_Scylla_Driver/notes/notes-app/README.md

```diff
@@ -1,20 +1,27 @@
 
 Demo Notes application to work on top of Rest Rust API server for Scylla for Notes application
 
+Warning - adding new notes is not implemented yet! (the + button doesn't work!)
+Just viewing and scrolling through existing ones works.
+Use REST API / postman (sample in rust server) to add new notes.
```

Comment:
LOL! I'll open an issue for some Angular/Ionic contributor finish this sample app.